mempool/clist_mempool.go

```diff
@@ -407,17 +416,17 @@ func (mem *CListMempool) RemoveTxByKey(txKey types.TxKey) error {
 }
 
 func (mem *CListMempool) isFull(txSize int) error {
-	var (
-		memSize  = mem.Size()
-		txsBytes = mem.SizeBytes()
-	)
+	memSize := mem.Size()
+	txsBytes := mem.SizeBytes()
+	recheckFull := mem.recheck.recheckFull.Load()
```

Comment:
```suggestion
	recheckFull := mem.config.Recheck && mem.recheck.recheckFull.Load()
```

Reply:
I don't think it's necessary... if `Recheck` is disabled in config, it will never be fired and so, `recheckFull` can never become full.